### PR TITLE
Added the Kubernetes scheduler to the component table

### PR DESCRIPTION
**Pull request by bergerhoffer**
Preview: https://deploy-preview-32184--osdocs.netlify.app/openshift-enterprise/latest/architecture/control-plane.html#defining-masters_control-plane

Makes updates from #32183, specific to 4.5.

### CODE REVIEW

File: modules/architecture-machine-roles.adoc [hunk 1]
```diff
@@ -44,15 +44,20 @@ Services that fall under the Kubernetes category on the master include the Kuber
...
 //etcd can be optionally configured for high availability, typically deployed with 2n+1 peer services.
+
 |Controller Manager Server
 |The Controller Manager Server watches etcd for changes to objects such as
 replication, namespace, and serviceaccount controller objects, and then uses the
 API to enforce the specified state. Several such processes create a cluster with
 one active leader at a time.
+
+|Kubernetes scheduler
+|The Kubernetes scheduler watches for newly created pods and selects the best node for the pod to run on.
```
Comment (skrthomas) [inline modules/architecture-machine-roles.adoc:60]: Same preposition de-clutter suggestion at the end of this sentence, "and selects the best node to run the pod."

**Review by skrthomas — COMMENTED** [review]
Same suggestion as other PR, but LGTM! Good job 💯 🥇

**Comment by bergerhoffer** [comment]
@kasturinarra Can you please QE review this update?

Preview: https://deploy-preview-32184--osdocs.netlify.app/openshift-enterprise/latest/architecture/control-plane.html#defining-masters_control-plane

Same update as #32183, but specific to 4.5 .

File: modules/architecture-machine-roles.adoc [hunk 2]
```diff
@@ -44,15 +44,20 @@ Services that fall under the Kubernetes category on the master include the Kuber
...
 //etcd can be optionally configured for high availability, typically deployed with 2n+1 peer services.
+
 |Controller Manager Server
 |The Controller Manager Server watches etcd for changes to objects such as
 replication, namespace, and serviceaccount controller objects, and then uses the
 API to enforce the specified state. Several such processes create a cluster with
 one active leader at a time.
+
+|Kubernetes scheduler
+|The Kubernetes scheduler watches for newly created pods and selects the best node to host the pod.
```
[the remark this answers is not in the document]
Comment (kasturinarra) [reply modules/architecture-machine-roles.adoc]: Thanks !!

**Pull request merged**